content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a month of busy development. Time to get back to testing.
+
+The mayor changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
+  - Improved system integration with `openra://` server join URLs and instant `.orarep` playback.
+  - The Dune 2000 mod got building upgrades which re-creates the tech tree of the original.
+  - Several enhanced classic missions have been added to the Tiberian Dawn mod.
+  - In the Red Alert mod you will now be notified when enemy units appear in your sight.
+
+We discovered a packaging problem in our last stable Windows build. This playtest should increase performance up to 30% on Windows.
+Also general performance improvements have been integrated that affect all platforms. Kudos to [RoosterDragon](https://github.com/RoosterDragon) for his continuous effort to optimize our source code.
+
+The refined system integration brings several new polish:
+
+  - The hardware cursors will now always work on Microsoft Windows.
+  - On Mac the dialogs won't hijack finder and the dock icon won't get lost upon restarting. Minimum supported OS X version is now 10.6.
+  - Linux AppStream compatible software centers will now show a nice preview.
```

Comment:
This would make a nice screenshot.  How about including a downscaled version of https://cloud.githubusercontent.com/assets/756669/7556248/5de9bafc-f769-11e4-8e92-d927c482a9d9.png ?